requirements-build.txt

```diff
@@ -1,2 +1,2 @@
-build==1.2.1
-hatchling==1.25.0
+build==1.2.2.post1
```

Comment:
should it be `.post1` ? or should be just do 1.2.2? or .3?

Reply:
https://pypi.org/project/build/

`.post1` looks right